desktop/src/navigator.rs

```diff
@@ -113,14 +106,33 @@ impl NavigatorBackend for ExternalNavigatorBackend {
     }
 
     fn fetch(&self, url: &str, _options: RequestOptions) -> OwnedFuture<Vec<u8>, Error> {
-        // Load from local filesystem.
-        // TODO: Support network loads, honor sandbox type (local-with-filesystem, local-with-network, remote, ...)
-        let mut path = self.relative_base_path.clone();
-        path.push(url);
-
-        Box::pin(async move { fs::read(path).map_err(Error::NetworkError) })
+        // TODO: honor sandbox type (local-with-filesystem, local-with-network, remote, ...)
+        let full_url = self.movie_url.clone().join(url).unwrap();
+
+        match full_url.scheme() {
+            "file" => Box::pin(async move { fs::read(full_url.to_file_path().unwrap()).map_err(Error::NetworkError) }),
+            _ => {
+                let mut client_builder = reqwest::Client::builder();
+                if let Some(proxy) = self.http_proxy.to_owned() {
+                    client_builder = client_builder.proxy(reqwest::Proxy::http(proxy.as_str()).unwrap())
+                }
```

Comment:
The Proxy constructors accept Url, and we can avoid the unwrap:
```
                if let Some(proxy) = self.http_proxy.to_owned() {
                    match reqwest::Proxy::http(proxy) {
                        Ok(proxy) => client_builder = client_builder.proxy(proxy),
                        Err(e) => log::error!("Unable to set HTTP proxy: {}", e),
                    }
                }
```
(the error should probably not happen, but just in case, it's better to display an error and recover gracefully than panic)